packages/perspective/test/js/pivots.spec.js

```diff
@@ -2355,25 +2355,21 @@ const std = (nums) => {
             table.delete();
         });
 
-        test("['z'] only, datetime column", async function () {
+        test("['z'] only, datetime column", async function ({ page }) {
```

Comment:
This `page` can be removed, I believe, as it was used for debugging. I can push an update.